template.yaml

```diff
@@ -633,7 +728,7 @@ Resources:
     Type: AWS::DynamoDB::Table
     Properties:
       # checkov:skip=CKV_AWS_28: Point in time recovery is not necessary for this table.
-      TableName: !Sub "f2f-front-sessions-${Environment}"
+      TableName: !Sub "${AWS::StackName}-sessions-${Environment}"
```

Comment:
this is now covered by 476, so shouldn't be changed here

Reply:
removed